Backlog.md User Story Review

DSF256 opened the pull request:
I went ahead and converted all of the extra features in Backlog.MD into user stories per the project instructions.  They're formatted similarly to the Features.MD file that's also in here.  Let me know what you think of them or if any other changes need to be made.

I also went in and checked off (or, rather, $$$'d off) some of the tasks that we have completed.

DSF256 committed:
Converted backlog.md to a bunch of user stories.

DSF256 committed:
Oh, so those breaks were there for a reason... ._.

NotHacker101 reviewed (commented):
i think user stories need to have better explain why we doing it.

DSF256 committed:
Updated and expanded instruction file

The instruction file should tell the player everything they need to know now, compared to the cursory (if that) overview the previous version did.

mbagwell reviewed (approved):
Thankyou for deleting the $$$ signs. I still do not know why we had those.

DSF256 commented:
Ah, those $$$ just mean that the task has been completed.  Think of it as a sort of check mark.

DSF256 committed:
Merge branch 'master' into jth_backlog_check
